snap/snapcraft.yaml

```diff
@@ -63,6 +63,12 @@ parts:
   spark8t-conf:
     plugin: dump
     source: snap/local
+    organize:
+      spark8t/spark-defaults.conf: etc/spark8t/spark-defaults.conf
+      spark/podTemplate.yaml: opt/spark/conf/podTemplate.yaml
+    stage:
+      - etc/spark8t/spark-defaults.conf
+      - opt/spark/conf/podTemplate.yaml
```

Comment:
Instead of "hacking" `podTemplate.yaml` into another place than the original `local` structure -- don't we wanna have a clear sturcture on `local`, precisely reflecting the delivery (I mean packed snap)? I think it would be both more intuitive and clean.

Practically what I mean is to supress all the new code above, and 
```
mkdir snap/local/spark/conf/
git mv snap/local/spark/podTemplate.yaml snap/local/spark/conf/podTemplate.yaml
```

Reply:
I don't think the path on "local" matters too much, given the organize flag. I kinda avoided creating very nested directories (it would be /snap/local/opt/spark/conf, but yes, it is clearer to map to the same structure. And we don't pay per number of directories, do we?

Reply:
Yeah, generally I agree to keep structures as flat as possible.
But here I rather prefer to keep consistency between "code" (repo) structure, and the actual deployment -- and use the `dump` plugin with "no additional magic" but only copying a clear, consistent structure over.